packages/wallet/src/KeyManagement/types.ts

```diff
@@ -187,8 +188,15 @@ export interface KeyAgent {
    * @throws AuthenticationError
    */
   exportRootPrivateKey(): Promise<Cardano.Bip32PrivateKey>;
+  /**
+   * @throws AuthenticationError
+   */
+  exportKeyPair(fullDerivationPath: number[]): Promise<KeyPair>;
```

Comment:
Let's not add this to KeyAgent interface. Instead, just on InMemoryKeyAgent. Then we won't need the commit in web-extension package.

Reply:
@mkazlauskas Isn't it better to keep it because if you use the wrong key agent e.g. LedgerKeyAgent we will throw a nice error that the operation is not supported by that key agent. We will also need those methods in other key agents because voting keys will be derived on a different way for HW ( we will see how ) or we will need to change approach once CIP proposal is approved?